backend/package.json

```diff
@@ -1,26 +1,63 @@
 {
-  "name": "pizza-sync",
+  "name": "nest-typescript-starter",
   "version": "1.0.0",
```

Comment:
this is a big change no version update?
(Really really useful comment 😄 )

Reply:
Yeah, it desserves at least a 2.0.0 version ! :)

Reply:
Bumped to 2.0.0! thx